AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -14,19 +17,18 @@
 // The group Id for storage.
 static NSString *const kMSGroupId = @"Identity";
 
+// The path component of Identity for configuration.
+static NSString *const kMSIdentityPathComponent = @"identity";
+
+// The Identity config file name.
+static NSString *const kMSIdentityConfigFilename = @"config.json";
+
 // Singleton
 static MSIdentity *sharedInstance = nil;
 static dispatch_once_t onceToken;
 
-// Authentication URL formats.
-static NSString *const kMSAuthorityFormat = @"https://login.microsoftonline.com/tfp/%@.onmicrosoft.com/%@";
-static NSString *const kMSAuthScopeFormat = @"https://%@.onmicrosoft.com/%@/user_impersonation";
-
-// Configuration (Temporary). Fill constant values for your backend.
-static NSString *const kMSTenantName = @"";
-static NSString *const kMSPolicyName = @"";
-static NSString *const kMSAPIIdentifier = @"";
-static NSString *const kMSClientId = @"";
+// Lock object for synchronization.
+static NSObject *const lock = @"lock";
```

Comment:
variable naming- shouldn't it be `kLock` or `sLock` or something? it's a static const

Reply:
`kLock` 🕐 ;)

Reply:
This variable follows the naming convention for static variables (not const). We can drop the `const` from there.

Reply:
Not a big deal to me, was just wondering. I thought we had a prefix or something so it doesn't look like a local variable but it sounds like I was wrong